packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfControlLibrary-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": ["Common", "WPF"],
+  "name": "WPF User Control Library",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A project for creating a user control library for .NET Core WPF Applications",
+  "groupIdentity": "Microsoft.Common.WPF.Control",
+  "precedence": "3000",
+  "identity": "Microsoft.Common.WPF.Control.CSharp.3.0",
+  "shortName": "wpfcontrollib",
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "sourceName": "Company.WpfControlLibrary",
+  "preferNameDirectory": true,
+  "symbols": {
+    "TargetFrameworkOverride": {
+      "type": "parameter",
+      "description": "Overrides the target framework",
+      "replaces": "TargetFrameworkOverride",
+      "datatype": "string",
+      "defaultValue": ""
+    },
+    "Framework": {
+      "type": "parameter",
+      "description": "The target framework for the project.",
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "netcoreapp3.0",
+          "description": "Target netcoreapp3.0"
+        }
+      ],
+      "replaces": "netcoreapp3.0",
+      "defaultValue": "netcoreapp3.0"
+    },
+    "langVersion": {
+      "type": "parameter",
+      "datatype": "text",
+      "description": "Sets langVersion in the created project file",
+      "defaultValue": "",
+      "replaces": "$(ProjectLanguageVersion)"
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
+    },
+    "skipRestore": {
+      "type": "parameter",
+      "datatype": "bool",
+      "description": "If specified, skips the automatic restore of the project on create.",
+      "defaultValue": "false"
+    }
+  },
+  "primaryOutputs": [
+    { "path": "Company.WpfControlLibrary.csproj" },
+    {
+      "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "path": "UserControl1.xaml"
```

Comment:
`Usercontrol1.xaml.cs`  should be included in the `primaryOutputs`, and `UserControl1.xaml` should be focused by default in VS. 

In Blend, only `UserControl1.xaml` should be opened.

Reply:
It is not possible to control template behavior in this way today in the .NET Core templating engine